monet_cyclegan_tutorial.ipynb

```diff
@@ -104,8 +104,14 @@
     "        record = tf.io.parse_single_example(record, tfrecord_format)\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #221.            up_sample(512, 4, apply_dropout=True), # (bs, 8, 8, 1024)
</pre><p>worth adding the notes (of layer sizes) for the submission</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/itayL1/deep-rl-final-project/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>